### PR TITLE
additions to requirements.txt when trying proto producer

### PR DESCRIPTION
**Pull request by dbwrangler**

### CODE REVIEW

**Comment by dbwrangler** [comment]
[clabot:check]

File: examples/requirements.txt [hunk 1]
```diff
@@ -7,3 +7,5 @@ pyrsistent==0.16.1;python_version<"3.0"
 pyrsistent;python_version>"3.0"
 jsonschema
 protobuf
+requests
```
Comment (edenhill) [inline examples/requirements.txt:10]: I'm thinking it should perhaps depend on confluent-kafka[avro] (et.al) instead to pull in these secondary dependencies?

Comment (dbwrangler) [reply examples/requirements.txt:10]: You mean update requirements with confluent-kafka[avro] instead? TBH would be nice to not have any avro references (other diff is fastavro, which seems to be a SchemaRegistryClient dependency),though I understand non-avro options are a relatively recent addition.

Comment (edenhill) [reply examples/requirements.txt:10]: The examples should depend on confluent-kafka[avro], not their respective secondary dependencies.

**Pull request closed without merging**